Gems/LyShine/Code/Source/Sprite.cpp

```diff
@@ -751,7 +758,7 @@ bool CSprite::DoesSpriteTextureAssetExist(const AZStd::string& pathname)
 
     // Check if the texture asset exists
     const AZStd::string cacheRelativePath = AZStd::string::format("%s.%s", pathname.c_str(), streamingImageExtension);
-    bool textureExists = CheckIfFileExists(/* sourceRelativePath = */ pathname, cacheRelativePath);
```

Comment:
Is this a carb specific engine change?

Reply:
This was patch by @aefimov3 , and I decided that substituting original `spritePath` with `pathname` was a mistake.

Reply:
I have a code piece that works locally that loads the sprites without engine patches. I sent you the code, please test it. It uses original lua scripts.